@ndlib/gatsby-theme-marble/src/components/Shared/ActionButtonGroup/DownloadButton/DownloadModalContent/DownloadCitation/index.js

```diff
@@ -1,5 +1,12 @@
-const DownloadCitation = ({ iiifManifest }) => {
-  return null
+import React from 'react'
+import PropTypes from 'prop-types'
+const DownloadCitation = ({ marbleItem }) => {
+  return marbleItem.citation ? <div>{marbleItem.citation}</div> : null
```

Comment:
Let's make sure we're not throwing away the styling work you already did, and ensure we're including the Title and extra explanatory text.

```
return marbleItem.citation ? (
  <div sx={sx.metadata}>	
    <Styled.h2 sx={sx.header}>{t('text:actionGroup.citationHead')}</Styled.h2>	
        {t('text:actionGroup.citationBody')}	
    <div className='citation' sx={sx.citation}>{marbleItem.citation}</div>	
  </div>
) : null
```
If we do the above, make sure you're including and calling the `useTranslation` function and also the sx.js file you moved.